### PR TITLE
Added privacy policies and enhanced UI for better user experience

### PR DESCRIPTION
**Pull request by akashdabhi03**
1. Added a toast for downloading Learning module and Library app
2. Added a percentage text on the progress bar and synced the progress bar with percentage value
3. Added a toast which is shown when Book Viewer is not installed. Additionally, the user is redirected to Google Play Store after the toast is shown

### CODE REVIEW

File: launcher/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -7,4 +7,5 @@
     <color name="colorCreditText">#f7f7f7</color>
     <color name="coinNumberText">#FDDB68</color>
     <color name="coinNumberTextDisabled">#D9D9D9</color>
+    <color name="black">#000000</color>
```
Comment (ankitk20) [inline launcher/app/src/main/res/values/colors.xml]: Remove this

File: mainapp/proj.android-studio/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1 +1,3 @@
-<resources></resources>
+<resources>
+    <string name="content_extraction" translatable="false">One time content extraction is in progress</string>
```
Comment (ankitk20) [inline mainapp/proj.android-studio/app/src/main/res/values/strings.xml]: Shouldn't this be in Hindi?

File: launcher/app/src/main/java/com/maq/xprize/kitkitlauncher/hindi/ToolsActivity.java [hunk 3]
```diff
@@ -294,7 +295,7 @@ public void onClick(View v) {
                             logger.logEvent("ToolsActivity", "start_writing_board", "", 0);
                         } catch (Exception e) {
                             startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://play.google.com/store/apps/details?id=" + packageName)));
-                            /*Toast.makeText(getApplicationContext(), "Error", Toast.LENGTH_SHORT).show();*/
+                            Toast.makeText(ToolsActivity.this, "Click the install button to install Writing Board app", Toast.LENGTH_LONG).show();
```
Comment (ankitk20) [inline launcher/app/src/main/java/com/maq/xprize/kitkitlauncher/hindi/ToolsActivity.java]: We don't need this

File: launcher/app/src/main/res/values/colors.xml (hunk 1 above)
Comment (akashdabhi03) [reply launcher/app/src/main/res/values/colors.xml]: Removed

File: mainapp/proj.android-studio/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (akashdabhi03) [reply mainapp/proj.android-studio/app/src/main/res/values/strings.xml]: Translated to Hindi

File: launcher/app/src/main/java/com/maq/xprize/kitkitlauncher/hindi/ToolsActivity.java (hunk 3 above)
Comment (akashdabhi03) [reply launcher/app/src/main/java/com/maq/xprize/kitkitlauncher/hindi/ToolsActivity.java]: Removed

File: library/app/src/main/java/utils/Zip.java [hunk 4]
```diff
@@ -42,10 +56,23 @@ public void unzip(String extractPath) throws IOException {
...
+            progressBar.setProgress(percent);
+            final int finalPercent = percent;
+            zipActivity.runOnUiThread(new Runnable() {
+                public void run() {
+                    // Show the percentage value on progress bar
+                    percentText.setText(finalPercent + " %");
+                }
+            });
+
             ZipEntry zipEntry = zipEntries.nextElement();
```
[the remark this answers is not in the document]
Comment (akashdabhi03) [reply library/app/src/main/java/utils/Zip.java]: Done

File: library/app/src/main/java/com/maq/xprize/kitkitlibrary/english/SplashScreenActivity.java [hunk 5]
```diff
@@ -103,24 +88,22 @@ public void unzipFile() {
             ActivityCompat.requestPermissions(this, new String[]{Manifest.permission.WRITE_EXTERNAL_STORAGE}, 99);
             File file = new File(filePath);
             ZipFile zipFile = new ZipFile(file);
-            Zip _zip = new Zip(zipFile);
+            Zip _zip = new Zip(zipFile, this);
```
[the remark this answers is not in the document]
Comment (akashdabhi03) [reply library/app/src/main/java/com/maq/xprize/kitkitlibrary/english/SplashScreenActivity.java]: Done

**Pull request merged**